modules/ROOT/pages/to-download-and-install-studio-wx.adoc

```diff
@@ -0,0 +1,58 @@
+= Downloading and Installing Anypoint Studio (Windows)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Prerequisites
+
+* Ensure that you have the Java SE JDK 8 installed and configured.
+* Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio/previous[Download Site].
+
+
+After downloading and installing the JDK, ensure that your JDK installation is properly configured within the `JAVA_HOME` environment variable:
+
+. In Search, search for `System (Control Panel)` and select it.
+. Click *Advanced System Settings*.
+. Click *Environment Variables*.
+. In *System Variables*, look for the PATH environment variable:
+** If `PATH` is visible, select it and click *Edit* to navigate to *Edit System Variable*.
+** If `PATH` is not visible, click *New* to navigate to *New System Variable*.
+. Ensure that the `/bin` directory of your JDK 1.8.0 installation is the first item in your `PATH` environment variable.
+. Click *OK*.
+
+== Installation
+
+. Extract the Windows installation file into the `C:\` root folder.
+** If you are using a Windows antivirus application, ensure that the `plugin/` and `features/` directories are under the _trusted_ category.
```

Comment:
Should this be indented to be a bullet under the number? If it renders as you wish it to, ignore this comment.